### PR TITLE
BC-13_restyle_app

### PR DESCRIPTION
**Pull request by dblaichinger**

### CODE REVIEW

File: resources/views/config.blade.php [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                $delete.addEventListener('click', () => {
+                    $notification.parentNode.removeChild($notification);
+                });
+            });
+        });
+
+        document.querySelector("#config-form").addEventListener("submit", function(e){
+            document.querySelector('.loader-wrapper').style.display = 'inline';
+            document.querySelector('.save-text').style.display = 'none';
+        });
```
[the remark this answers is not in the document]
Comment (TheKeymaster) [reply resources/views/config.blade.php:108]: The benefit is that we can use ES6 and do not have to worry about older browsers.

File: public/js/app.js [hunk 2]
```diff
@@ -0,0 +1 @@
+window.bootstrap = require('bootstrap/dist/js/bootstrap.bundle.js');
```
Comment (TheKeymaster) [inline public/js/app.js:1]: Why are you assigning this to the `window`?

**Review by TheKeymaster — CHANGES_REQUESTED** [review]

File: resources/views/config.blade.php [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+                    <label for="shopkey" class="col-sm-3 col-md-2 col-form-label">Shop key</label>
+                    <div class="col-sm-9 col-md-10">
+                        <input name="shopkey"
+                               class="form-control"
+                               aria-describedby="shopkeyHelp"
+                               value="@isset($shopkey){{ $shopkey }}@endisset"
+                        >
+                        @empty($shopkey)
+                            <div id="shopkeyHelp" class="form-text">
+                                Please <a href="https://www.findologic.com/en/contact/" target="_blank">contact us</a>
```
[the remark this answers is not in the document]
Comment (dblaichinger) [reply resources/views/config.blade.php:45]: The BigCommerce backend offers translations for users but EN is enough for the app. No review feedback regarding translations.
Screen Shot 2021-05-03 at 19 07 20

File: public/js/app.js (hunk 2 above)
Comment (dblaichinger) [reply public/js/app.js:1]: I had really issues setting this up, no `import 'bootstrap'` or similar works. So if followed this recommendation: https://stackoverflow.com/a/65260998/1058552

**Review by TheKeymaster — APPROVED** [review]

**Pull request merged**